packages/prosess-vilkar-opptjening-oms/src/components/OpptjeningVilkarAksjonspunktPanel.tsx

```diff
@@ -95,6 +100,20 @@ export const OpptjeningVilkarAksjonspunktPanelImpl = ({
   const isOpenAksjonspunkt = aksjonspunkter.some(ap => isAksjonspunktOpen(ap.status.kode));
   const originalErVilkarOk = isOpenAksjonspunkt ? undefined : vilkarUtfallType.OPPFYLT === status;
 
+  const finnesOpptjeningsaktiviteterVidSkjeringstidspunkt = (): boolean => {
+    if(erPleiepenger){
+      return opptjeninger.some(opptjening => {
+        const skjeringstidspunkt = dayjs(opptjening.fastsattOpptjening.opptjeningTom).add(1, 'days');
+
+        return opptjening.opptjeningAktivitetList.some(opptjeningAktivitet =>
+          // Siste argument ("[]") til isBetween inkluderer start og sluttdato
+          dayjs(skjeringstidspunkt).isBetween(opptjeningAktivitet.opptjeningFom, opptjeningAktivitet.opptjeningTom, null, "[]")
+        );
+      });
```

Comment:
Er det nødvendig å dra inn eit nytt js-bibliotek her? Dette kan håndteres med moment. Og etter nærmere ettertanke så trur eg det er greit å sjekke på opptjeningTom og ikkje dagen før. Vi håndterer dette i beregning.

```suggestion
      return opptjeninger.some(opptjening => {
        return opptjening.opptjeningAktivitetList.some(opptjeningAktivitet =>
          !moment(opptjening.fastsattOpptjening.opptjeningTom).isAfter(moment(opptjeningAktivitet.opptjeningTom))
        );
      });
```